src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,58 +1,377 @@
+                  intake.intake();
+                }))
+            .onFalse(intake.retract());
+        // .onFalse(new InstantCommand(
+        //   () -> intake.retract()));
+
+
+
+    // 'a' button outtakes note
+    operatorController
+        .rightBumper()
+        .whileTrue(new InstantCommand(() -> intake.outtake()))
+        .onFalse(new InstantCommand(() -> intake.zero()));
+
+    // driver.y().onTrue(autonomous.SHOOT_MOBILITY());
+
+    /*// Left bumper shoots Speaker, now shootSpeaker also includes moving intake also.
+    operator
+        .leftTrigger()
+        .whileTrue(shooter.shootSpeaker())
+        .onFalse(
+            new InstantCommand(
+                () -> {
+                  shooter.zero();
+                  intake.zero();
+                }));
+
+    // Left joystick CLICK shoots amp, also moves intake using one button
+    operator
+        .rightTrigger()
+        .whileTrue(shooter.shootAmp())
+        .onFalse(
+            new InstantCommand(
+                () -> {
+                  shooter.zero();
+                  intake.zero();
+                }));*/
+
+    operatorController
+        .povUp()
+        .whileTrue(new InstantCommand(() -> shooter.setSpeedReverse()))
+        .onFalse(
+            new InstantCommand(
+                () -> {
+                  shooter.zero();
+                 // intake.zero();
+                }));
+
+    operatorController
+    .povDown()
+    .whileTrue(new InstantCommand(() -> shooter.setSpeakerSpeed()))
+    .onFalse(
+        new InstantCommand(
+            () -> {
+                shooter.zero();
+                //intake.zero();
+            }
+        )
+    );
+
+    operatorController
+    .povLeft()
+    .whileTrue(new InstantCommand(() -> testShooterAndIntake()))
+    .onFalse(
+        new InstantCommand(
+            () -> {
+                shooter.zero();
+                intake.zero();
+            }
+        )
+    );
+
+    operatorController
+        .a()
+        .whileTrue(
+            new InstantCommand(
+                () -> {
+                  arm.goToClimbDownPos();
+                }));
+
+    operatorController
+        .b()
+        .whileTrue(ampButtonBinding())
+        .onFalse(
+            new ParallelCommandGroup(
+                new InstantCommand(() -> arm.goToShootPos()),
+                new InstantCommand(() -> intake.zero()),
+                new InstantCommand(() -> shooter.zero())));
+
+    operatorController
+        .y()
+        .whileTrue(
+            new InstantCommand(
+                () -> {
+                  arm.goToClimbUpPos();
+                }));
+
+    operatorController
+        .x()
+        .whileTrue(shootButtonBinding())
+        .onFalse(
+            new ParallelCommandGroup(
+                new InstantCommand(() -> arm.goToShootPos()),
+                new InstantCommand(() -> intake.zero()),
+                new InstantCommand(() -> shooter.zero())));
+
+    // drive.setDefaultCommand(
```

Comment:
Lots of dead code to cleanup here.

Reply:
I'll remove all the unneeded comments from RobotContainer.java.

Reply:
I see, that makes sense. That change has now been implemented.